app/components/tables/headers/sort.js

```diff
@@ -21,7 +21,7 @@ export default class extends Component {
     super.didInsertElement(...arguments);
     if (this.sorts && this.sorts[0] && this.sorts[0].valuePath === this.column.valuePath) {
       this.setProperties({
-        sortBy  : kebabCase(this.sorts[0].valuePath), // Ensures field names are server compatible with sort
+        sortBy  : this.sorts[0].valuePath === 'identifier' ? 'name' : kebabCase(this.sorts[0].valuePath), // Ensures field names are server compatible with sort
```

Comment:
Why? Event is not only table using the sorting

Reply:
yes, you're right. The thing is, when I checked I didn't find any other sortable field with valuePath of identifier, so, that would work for that case. But I just fount other one

Reply:
What I can do is at `ember-table-route.js` like this

![image](https://user-images.githubusercontent.com/47331628/66839461-5a274c80-ef5e-11e9-8e51-421df717bee7.png)

because, params will only have a `event_state` if it comes from `events/` url.

Or

I can pass a new argument with the text `event` to `applySortFilters` at `routes/events/list.js` and verify it at `applySortFilters`

Reply:
@JoaoJesus94 You can see #3488, Similar kind of change is involved in following PR. Use it as a reference and solve the following. Also changing `ember-table-route` will affect every ember-table we are using in the web-app, and if you will pass a new argument in `routes/events/list.js` it will fail as the sort filters is getting called from `ember-table-route`